package.json

```diff
@@ -103,7 +103,25 @@
         "description": "",
         "visible": true,
         "subPermissions": [
-          "finance.all",
+          "finance.awaiting-payment.item.post",
```

Comment:
I don't think we post anything to it from UI, according to https://wiki.folio.org/display/FOLIJET/Proposed+Permission+Set+Guidelines#ProposedPermissionSetGuidelines-Using*.allPermissions
_use just the permissions actually needed_

Reply:
Correct, removed them